pages/about/privacy-policy.vue

```diff
@@ -1,9 +1,36 @@
 <template>
-    <p>PRIVACY POLICY</p>
+    <div class="px-64 py-24">
```

Comment:
Please use the [`I18nPageWrapper`](https://github.com/pycontw/pycontw-2021/blob/main/components/core/i18n/PageWrapper.vue) for the styling.